infer panic in queue/unique_queue_disk_channel.go

```
panic: interface conversion: queue.Queue is *queue.LevelUniqueQueue, not *queue.LevelQueue

goroutine 222 [running]:
code.gitea.io/gitea/modules/queue.(*PersistableChannelUniqueQueue).Run.func4(0xc002c487e0)
        /home/lunny/shares/data/Projects/gitea/gitea/modules/queue/unique_queue_disk_channel.go:191 +0x24d
created by code.gitea.io/gitea/modules/queue.(*PersistableChannelUniqueQueue).Run
        /home/lunny/shares/data/Projects/gitea/gitea/modules/queue/unique_queue_disk_channel.go:189 +0x290
```

--- a/modules/queue/unique_queue_disk_channel.go
+++ b/modules/queue/unique_queue_disk_channel.go
@@ -188,7 +188,7 @@ func (q *PersistableChannelUniqueQueue) Run(atShutdown, atTerminate func(func())
 		go q.internal.Run(func(_ func()) {}, func(_ func()) {})
 		go func() {
 			_ = q.internal.Flush(0)
-			log.Debug("LevelUniqueQueue: %s flushed so shutting down", q.internal.(*LevelQueue).Name())
+			log.Debug("LevelUniqueQueue: %s flushed so shutting down", q.internal.(*LevelUniqueQueue).Name())
 			q.internal.(*LevelUniqueQueue).Shutdown()
 			GetManager().Remove(q.internal.(*LevelUniqueQueue).qid)
 		}()

